ProjectLighthouse/Controllers/ScoreController.cs

```diff
@@ -49,20 +59,31 @@ public async Task<IActionResult> SubmitScore(int id)
             {
                 this.database.Scores.Add(score);
             }
+
             await this.database.SaveChangesAsync();
 
-            return this.Ok();
+            string myRanking = await GetScores(score.SlotId, score.Type, user);
+
+            return this.Ok(myRanking);
         }
 
+        [HttpGet("friendscores/user/{slotId:int}/{type:int}")]
+        public async Task<IActionResult> FriendScores(int slotId, int type)
```

Comment:
Since theres no async calls:

```suggestion
        public IActionResult FriendScores(int slotId, int type)
```